src/assets/data/balancers.json

```diff
@@ -0,0 +1,49 @@
+{
+    "cheatSheet": {
+        "title": "Balancers",
+        "icon": "Fast_splitter"
+    },
+    "data": {
+        "table": [
+            {
+                "name": "1-8 belt balancers",
+                "yellow": "yes",

```

Comment:
instead of yes and no, use a boolean true and false like
"yelow": true,

Reply:
Done https://github.com/deniszholob/factorio-cheat-sheet/pull/82/commits/c4e84588e3a4fc815aad008b929130a64e663c3f